Bring back "actionable and reproducible" verbiage in the issue template

therealkenc opened the pull request:
template preamble.  Reword to be less shouty.

Refresh issue label descriptions in the README. Back-reference
the issue template for tag need-repro.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/WSL?pullRequest=3733) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/WSL?pullRequest=3733) before we can accept your contribution.<br/><br/>:x: therealkenc [sign now](https://cla.opensource.microsoft.com/Microsoft/WSL?pullRequest=3733)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/WSL?pullRequest=3733) it.</sub>

therealkenc commented:
The intent here is to bring back the original "actionable and reproducible" verbiage we used to have in the issue template circa 2017. Also updated the README because we didn't actually have a description for tag 'need-repro'. As you've probably noticed the incidents of submissions that aren't actionable has been on a steady rise over time, and this is a better-late-than-never effort to clarify the tags in the hopes it might help.

[Note I had accidentally pushed these changes directly/unilaterally to the Microsoft/WSL remote on my first go - apologies for that. I cleaned up my divot; hopefully that commit and rollback was harmless and unnoticeable.]

gpotter2 commented on README.md:
```diff
@@ -1,34 +1,42 @@
   - Example: No internet connectivity in Bash [(#5)](https://github.com/Microsoft/WSL/issues/5)
 
-- **feature** – Denotes something that is not yet implemented.  The community should use our [User Voice](https://wpdev.uservoice.com/forums/266908-command-prompt-console-bash-on-ubuntu-on-windo/category/161892-bash) page for voting on which features everyone feels as the most important.  The team will take the User Voice page as input in deciding what to work on next.
+- **feature** – Denotes something understood as not working and is not yet implemented. The community should use our [User Voice](https://wpdev.uservoice.com/forums/266908-command-prompt-console-bash-on-ubuntu-on-windo/category/161892-bash) page for voting on which features everyone feels as the most important.  The team will take the User Voice page as input in deciding what to work on next.
```
You might be missing a "that" ?

benhillis commented:
Thanks Ken, good changes.
